### PR TITLE
DATA: Add Post Impression Model & Migration

### LINKED ISSUE #166: DATA: Post Impressions Model & Migrations
Design and run the migrations for the following model:

```csharp
public class PostImpression
{
     public Guid PostId {get; set;}
     public Post Post {get; set;}
     
     public Guid ProfileId {get; set;}
     public Profile Profile {get; set;}
     
     public DateTimeOffset CreatedDate {get; set;}
     public DateTimeOffset UpdatedDate {get; set;}
     
     public ImpressionType Impression {get; set;}
}

public enum ImpressionType
{
     None,
     Like,
     Dislike
}
```

Helpful videos to show how to finalize this task:
https://www.youtube.com/watch?v=6sD5zz9BuOk
https://www.youtube.com/watch?v=XoDwJ-dvhI0

https://github.com/hassanhabib/The-Standard/blob/master/0.%20Introduction/0.0%20Purposing%2C%20Modeling%20%26%20Simulation.md

This task is assigned to @tsatay

### PR DESCRIPTION
**Pull request by ysnarafat**
close #166

### CODE REVIEW

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]
Are we missing the Reference.cs file?

**Comment by ysnarafat** [comment]
> Are we missing the Reference.cs file?

I assumed it has to be added later likewise some other PR

**Comment by ShriHumrudha** [comment]
@ysnarafat Take these references:

https://github.com/hassanhabib/Taarafo.Core/pull/38
https://www.youtube.com/watch?v=XoDwJ-dvhI0

**Review by ShriHumrudha — APPROVED** [review]

File: Taarafo.Core/Migrations/20220411061952_AddPostImpressionModel.cs [hunk 1]
```diff
@@ -13,12 +13,11 @@ namespace Taarafo.Core.Migrations
     public partial class AddPostImpressionModel : Migration
     {
         protected override void Up(MigrationBuilder migrationBuilder)
-        {
+        {
```
[the remark this answers is not in the document]
Comment (ysnarafat) [reply Taarafo.Core/Migrations/20220411061952_AddPostImpressionModel.cs:16]: Okay

**Pull request merged**